lib/route.method.js

```diff
@@ -139,6 +139,15 @@ RouteMethod.prototype.validate = function (validateFunc) {
     return this;
 };
 
+RouteMethod.prototype.notAllowed = function () {
+    this.options.notAllowed = true;
+    return this;
+};
+
+RouteMethod.prototype.isAllowed = function () {
```

Comment:
I see isAllowed and notAllowed, and they seem very different, and surprisingly so. notAllowed seems to be a setter with a default value. I would expect it to look something like setAllowed(false) or setNotAllowed() - the keyword "set" would let us know that this is a setter.